[06/27/18, 22:23:03:307] info: ╔═════════════════════════════════════════════════════════════════════════════════╗
[06/27/18, 22:23:03:308] info: ║      Slack 3.2.1, linux 4.10.0-38-generic on x64 at 06/27/18, 22:23:03:304      ║
[06/27/18, 22:23:03:308] info: ╚═════════════════════════════════════════════════════════════════════════════════╝
[06/27/18, 22:23:03:410] info: Initializing local storage instance at path: /home/nozomi/dotfiles/.config/Slack/local-settings.json
[06/27/18, 22:23:03:763] error: Desktop module calls failed to load
{
  "error": {
    "message": "libCallsCore.so: å±æãªãã¸ã§ã¯ããã¡ã¤ã«ãéãã¾ãã: ãã®ãããªãã¡ã¤ã«ããã£ã¬ã¯ããªã¯ããã¾ãã",
    "stack": "Error: libCallsCore.so: å±æãªãã¸ã§ã¯ããã¡ã¤ã«ãéãã¾ãã: ãã®ãããªãã¡ã¤ã«ããã£ã¬ã¯ããªã¯ããã¾ãã\n    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:172:20)\n    at Object.Module._extensions..node (module.js:671:18)\n    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:186:18)\n    at Module.load (module.js:561:32)\n    at tryModuleLoad (module.js:504:12)\n    at Function.Module._load (module.js:496:3)\n    at Module.require (module.js:586:17)\n    at require (internal/module.js:11:18)\n    at bindings (/usr/lib/slack/resources/app.asar/node_modules/@slack/slack-calls-prebuilt/modules/@slack/slack-calls/node_modules/bindings/bindings.js:81:44)\n    at Object.<anonymous> (/usr/lib/slack/resources/app.asar/node_modules/@slack/slack-calls-prebuilt/modules/@slack/slack-calls/index.js:1:192)"
  }
}
[06/27/18, 22:23:03:764] error: Desktop module calls failed to load
[06/27/18, 22:23:03:784] info: cld3::loadModule: loading cld3 module
[06/27/18, 22:23:03:804] info: cld3Loader::loadModule: running environment is NODE
[06/27/18, 22:23:03:805] info: cld3Loader::loadModule: constructed module object for runtime
[06/27/18, 22:23:03:984] info: cld3Loader::initializeRuntime: successfully initialized module
[06/27/18, 22:23:03:984] info: cld3Loader::loadModule: initialized wasm binary Runtime
[06/27/18, 22:23:03:995] info: SpellChecker:: updateSpellcheckerLanguage: Skip to set new language null, v2 spellchecker is not available
[06/27/18, 22:23:04:010] info: Main window preload script finished
{
  "readyTime": 999.721751
}
[06/27/18, 22:23:05:401] info: Store: SELECT_CHANNEL DB9T2S1C2
[06/27/18, 22:23:05:403] info: Store: UPDATE_TEAM_LOCALE
{
  "locale": "en-US",
  "teamId": "TB9PVC95H"
}
[06/27/18, 22:23:05:405] info: Store: UPDATE_TEAM_THEME
{
  "teamId": "TB9PVC95H",
  "theme": {
    "active_item": "#F79F66",
    "active_item_text": "#FFFFFF",
    "active_presence": "#F79F66",
    "badge": "#F15340",
    "column_bg": "#0D7E83",
    "hover_item": "#D37C71",
    "menu_bg": "#076570",
    "text_color": "#FFFFFF"
  }
}
[06/27/18, 22:23:05:414] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:07:466] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 3,
  "state": "loaded",
  "teamId": "T031FNTUT"
}
[06/27/18, 22:23:07:784] info: Store: UPDATE_SESSION_STATUS
{
  "hasValidSession": true,
  "teamId": "T031FNTUT"
}
[06/27/18, 22:23:07:831] error: Jun-27 22:23:07.831 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:07:843] error: 2018/6/27 22:23:07.843 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:08:814] info: Skipping request to provide hint content for channel for spellchecker
{
  "version": "V2"
}
[06/27/18, 22:23:09:560] info: Store: SELECT_CHANNEL C9Z4BU1C2
[06/27/18, 22:23:09:561] info: Store: UPDATE_TEAM_LOCALE
{
  "locale": "en-US",
  "teamId": "T031FNTUT"
}
[06/27/18, 22:23:09:562] info: Store: UPDATE_TEAM_THEME
{
  "teamId": "T031FNTUT",
  "theme": {
    "active_item": "#95BFEA",
    "active_item_text": "#FFFFFF",
    "active_presence": "#FFFF00",
    "badge": "#AAFFFF",
    "column_bg": "#40E0D0",
    "hover_item": "#48D1CC",
    "menu_bg": "#AAFFFF",
    "text_color": "#FFFFFF"
  }
}
[06/27/18, 22:23:10:398] error: Jun-27 22:23:10.398 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:10:404] error: 2018/6/27 22:23:10.403 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:12:459] error: Jun-27 22:23:12.459 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:12:464] error: 2018/6/27 22:23:12.463 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:13:923] error: Jun-27 22:23:13.922 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:13:925] error: 2018/6/27 22:23:13.925 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:15:693] error: Jun-27 22:23:15.693 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:15:698] error: 2018/6/27 22:23:15.698 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:18:494] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:20:069] error: Jun-27 22:23:20.069 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:20:077] error: 2018/6/27 22:23:20.760 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:21:816] error: Jun-27 22:23:21.816 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:21:820] error: 2018/6/27 22:23:21.820 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:23:865] error: Jun-27 22:23:23.865 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:23:869] error: 2018/6/27 22:23:23.868 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:24:877] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:27:857] error: Jun-27 22:23:27.857 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:27:874] error: 2018/6/27 22:23:27.874 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:28:798] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:33:299] info: Store: START_BACKGROUND_CLIENT
{
  "teamId": "TB47MRNBZ",
  "token": "xoxs-⋯"
}
[06/27/18, 22:23:33:721] info: Store: SET_BACKGROUND_CLIENT_FIELDS
{
  "status": "connected",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:23:33:727] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4001,
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:23:35:860] error: Jun-27 22:23:35.860 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:35:867] error: 2018/6/27 22:23:35.867 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:44:358] error: Jun-27 22:23:44.355 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:44:384] error: 2018/6/27 22:23:44.383 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:48:227] error: Jun-27 22:23:48.227 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:48:237] error: 2018/6/27 22:23:48.237 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:48:916] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:58:160] error: Jun-27 22:23:58.157 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:58:173] error: 2018/6/27 22:23:58.173 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:24:03:748] info: Store: START_BACKGROUND_CLIENT
{
  "teamId": "TB47MRNBZ",
  "token": "xoxs-⋯"
}
[06/27/18, 22:24:04:164] info: Store: SET_BACKGROUND_CLIENT_FIELDS
{
  "status": "connected",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:04:943] info: Store: SET_FULL_SCREEN
[06/27/18, 22:24:10:976] info: Store: SET_FULL_SCREEN
[06/27/18, 22:24:14:477] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4337,
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:14:499] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 5,
  "state": "unloaded",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:14:614] info: Store: SET_FULL_SCREEN
[06/27/18, 22:24:17:354] info: Store: SET_FULL_SCREEN
[06/27/18, 22:24:18:318] info: Store: SAVE_WINDOW_SETTINGS
{
  "isMaximized": false,
  "position": [
    574,
    106
  ],
  "size": [
    1024,
    768
  ]
}
[06/27/18, 22:24:20:143] error: Jun-27 22:24:20.143 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:24:20:145] error: 2018/6/27 22:24:20.145 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:24:20:361] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 5,
  "state": "loaded",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:20:913] info: Store: UPDATE_SESSION_STATUS
{
  "hasValidSession": true,
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:21:564] info: Store: SELECT_CHANNEL CB47MRRTR
[06/27/18, 22:24:21:568] info: Store: UPDATE_TEAM_LOCALE
{
  "locale": "en-US",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:21:580] info: Store: UPDATE_TEAM_THEME
{
  "teamId": "TB47MRNBZ",
  "theme": {
    "active_item": "#6698C8",
    "active_item_text": "#FFFFFF",
    "active_presence": "#94E864",
    "badge": "#78AF8F",
    "column_bg": "#303E4D",
    "hover_item": "#4A5664",
    "menu_bg": "#2C3849",
    "text_color": "#FFFFFF"
  }
}
[06/27/18, 22:24:39:858] error: Jun-27 22:24:39.858 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:24:39:861] error: 2018/6/27 22:24:39.861 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:25:36:904] error: Jun-27 22:25:36.904 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:25:36:916] error: 2018/6/27 22:25:36.916 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:25:39:968] error: Jun-27 22:25:39.967 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:25:39:976] error: 2018/6/27 22:25:39.976 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:25:42:986] error: Jun-27 22:25:42.985 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:25:42:994] error: 2018/6/27 22:25:42.994 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:25:46:930] error: Jun-27 22:25:46.929 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:25:46:939] error: 2018/6/27 22:25:46.939 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:26:06:263] error: Jun-27 22:26:06.262 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:26:06:273] error: 2018/6/27 22:26:06.273 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-hdsq:10443","stack":"Error: Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-hdsq:10443\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:26:24:957] error: Jun-27 22:26:24.956 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:26:24:968] error: 2018/6/27 22:26:24.967 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:27:05:006] error: Jun-27 22:27:05.005 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:27:05:012] error: 2018/6/27 22:27:05.120 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:27:14:897] error: Jun-27 22:27:14.896 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:27:14:905] error: 2018/6/27 22:27:14.904 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:27:34:082] error: Jun-27 22:27:34.082 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:27:34:085] error: 2018/6/27 22:27:34.840 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:27:42:004] error: Jun-27 22:27:42.004 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:27:42:007] error: 2018/6/27 22:27:42.700 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:27:57:931] error: Jun-27 22:27:57.931 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:27:57:937] error: 2018/6/27 22:27:57.936 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:03:861] error: Jun-27 22:28:03.860 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:03:869] error: 2018/6/27 22:28:03.868 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:09:025] error: Jun-27 22:28:09.024 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:09:034] error: 2018/6/27 22:28:09.340 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:18:298] error: Jun-27 22:28:18.298 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:18:304] error: 2018/6/27 22:28:18.304 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:20:943] error: Jun-27 22:28:20.943 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:20:946] error: 2018/6/27 22:28:20.946 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:24:943] error: Jun-27 22:28:24.943 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:24:951] error: 2018/6/27 22:28:24.951 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:30:960] error: Jun-27 22:28:30.959 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:30:966] error: 2018/6/27 22:28:30.965 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:37:169] error: Jun-27 22:28:37.169 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:37:177] error: 2018/6/27 22:28:37.176 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:40:120] error: Jun-27 22:28:40.119 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:40:132] error: 2018/6/27 22:28:40.131 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:43:003] error: Jun-27 22:28:43.002 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:43:008] error: 2018/6/27 22:28:43.800 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:48:077] error: Jun-27 22:28:48.076 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:48:079] error: 2018/6/27 22:28:48.790 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:57:072] error: Jun-27 22:28:57.072 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:57:075] error: 2018/6/27 22:28:57.750 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:29:07:949] error: Jun-27 22:29:07.948 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:29:07:960] error: 2018/6/27 22:29:07.957 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:29:11:929] error: Jun-27 22:29:11.929 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:29:11:938] error: 2018/6/27 22:29:11.938 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:29:14:985] error: Jun-27 22:29:14.985 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:29:14:999] error: 2018/6/27 22:29:14.999 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:29:19:011] error: Jun-27 22:29:19.011 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:29:19:017] error: 2018/6/27 22:29:19.160 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:29:26:093] error: Jun-27 22:29:26.092 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:29:26:100] error: 2018/6/27 22:29:26.100 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:29:34:105] error: Jun-27 22:29:34.100 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:29:34:113] error: 2018/6/27 22:29:34.112 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:29:51:922] error: Jun-27 22:29:51.922 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:29:51:928] error: 2018/6/27 22:29:51.928 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:30:19:991] error: Jun-27 22:30:19.990 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:30:20:004] error: 2018/6/27 22:30:20.300 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:30:42:907] error: Jun-27 22:30:42.906 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:30:42:915] error: 2018/6/27 22:30:42.915 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:12:987] error: Jun-27 22:31:12.987 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:12:995] error: 2018/6/27 22:31:12.995 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:22:901] error: Jun-27 22:31:22.901 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:22:911] error: 2018/6/27 22:31:22.910 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:25:931] error: Jun-27 22:31:25.930 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:25:939] error: 2018/6/27 22:31:25.938 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:28:899] error: Jun-27 22:31:28.899 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:28:908] error: 2018/6/27 22:31:28.907 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:33:983] error: Jun-27 22:31:33.982 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:33:989] error: 2018/6/27 22:31:33.989 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:41:907] error: Jun-27 22:31:41.907 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:41:915] error: 2018/6/27 22:31:41.915 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:48:009] error: Jun-27 22:31:48.008 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:48:020] error: 2018/6/27 22:31:48.200 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:50:845] error: Jun-27 22:31:50.845 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:50:857] error: 2018/6/27 22:31:50.856 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:54:916] error: Jun-27 22:31:54.916 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:54:925] error: 2018/6/27 22:31:54.924 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:58:939] error: Jun-27 22:31:58.938 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:58:947] error: 2018/6/27 22:31:58.947 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:32:01:984] error: Jun-27 22:32:01.984 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:32:01:991] error: 2018/6/27 22:32:01.991 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:32:04:975] error: Jun-27 22:32:04.974 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:32:04:984] error: 2018/6/27 22:32:04.983 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:32:08:933] error: Jun-27 22:32:08.933 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:32:08:950] error: 2018/6/27 22:32:08.949 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:32:11:935] error: Jun-27 22:32:11.935 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:32:11:938] error: 2018/6/27 22:32:11.938 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:32:17:993] error: Jun-27 22:32:17.992 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:32:18:003] error: 2018/6/27 22:32:18.300 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:32:27:925] error: Jun-27 22:32:27.925 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:32:27:932] error: 2018/6/27 22:32:27.932 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:32:52:951] error: Jun-27 22:32:52.951 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:32:52:960] error: 2018/6/27 22:32:52.959 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:33:52:978] error: Jun-27 22:33:52.977 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:33:53:005] error: 2018/6/27 22:33:53.400 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:34:54:963] error: Jun-27 22:34:54.963 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:34:54:966] error: 2018/6/27 22:34:54.966 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:35:57:064] error: Jun-27 22:35:57.063 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:35:57:070] error: 2018/6/27 22:35:57.700 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:36:54:041] error: Jun-27 22:36:54.041 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:36:54:045] error: 2018/6/27 22:36:54.450 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:37:55:963] error: Jun-27 22:37:55.963 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:37:55:971] error: 2018/6/27 22:37:55.971 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:38:57:978] error: Jun-27 22:38:57.977 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:38:57:980] error: 2018/6/27 22:38:57.980 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:40:00:300] error: Jun-27 22:40:00.300 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:40:00:307] error: 2018/6/27 22:40:00.307 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:40:46:279] error: Jun-27 22:40:46.279 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:40:46:282] error: 2018/6/27 22:40:46.281 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:41:48:040] error: Jun-27 22:41:48.040 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:41:48:045] error: 2018/6/27 22:41:48.450 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:42:40:643] error: Jun-27 22:42:40.642 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:42:40:651] error: 2018/6/27 22:42:40.651 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:43:43:068] error: Jun-27 22:43:43.067 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:43:43:077] error: 2018/6/27 22:43:43.760 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:44:44:933] error: Jun-27 22:44:44.933 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:44:44:936] error: 2018/6/27 22:44:44.936 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:45:47:014] error: Jun-27 22:45:47.013 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:45:47:020] error: 2018/6/27 22:45:47.200 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:46:59:334] error: Jun-27 22:46:59.333 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:46:59:343] error: 2018/6/27 22:46:59.342 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-jtit:10443","stack":"Error: Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-jtit:10443\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:48:01:077] error: Jun-27 22:48:01.077 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:48:01:079] error: 2018/6/27 22:48:01.790 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:49:03:141] error: Jun-27 22:49:03.141 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:49:03:153] error: 2018/6/27 22:49:03.152 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:49:15:936] error: Jun-27 22:49:15.936 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:49:15:942] error: 2018/6/27 22:49:15.941 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:49:25:915] error: Jun-27 22:49:25.915 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:49:25:919] error: 2018/6/27 22:49:25.918 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:49:30:933] error: Jun-27 22:49:30.933 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:49:30:935] error: 2018/6/27 22:49:30.935 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:49:36:945] error: Jun-27 22:49:36.945 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:49:36:956] error: 2018/6/27 22:49:36.955 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:49:42:842] error: Jun-27 22:49:42.841 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:49:42:849] error: 2018/6/27 22:49:42.849 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:49:47:894] error: Jun-27 22:49:47.894 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:49:47:897] error: 2018/6/27 22:49:47.897 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:49:59:910] error: Jun-27 22:49:59.910 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:49:59:918] error: 2018/6/27 22:49:59.918 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:50:31:037] error: Jun-27 22:50:31.036 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:50:31:049] error: 2018/6/27 22:50:31.440 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:51:33:022] error: Jun-27 22:51:33.021 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:51:33:027] error: 2018/6/27 22:51:33.270 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:52:45:345] error: Jun-27 22:52:45.345 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:52:45:351] error: 2018/6/27 22:52:45.351 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-jtit:10443","stack":"Error: Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-jtit:10443\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:53:47:022] error: Jun-27 22:53:47.022 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:53:47:026] error: 2018/6/27 22:53:47.250 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:54:49:031] error: Jun-27 22:54:49.031 SOCKET-MANAGER (T031FNTUT) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:54:49:039] error: 2018/6/27 22:54:49.390 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:55:31:723] info: Store: FOREGROUND_APP
[06/27/18, 22:55:31:769] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:31:770] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:31:804] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:32:749] info: Store: SAVE_WINDOW_SETTINGS
{
  "isMaximized": false,
  "position": [
    575,
    131
  ],
  "size": [
    1024,
    768
  ]
}
[06/27/18, 22:55:33:757] info: Store: MAIN_WINDOW_FOCUSED
[06/27/18, 22:55:34:367] info: Store: MAIN_WINDOW_FOCUSED
[06/27/18, 22:55:34:553] info: Store: SELECT_TEAM TB47MRNBZ
[06/27/18, 22:55:41:723] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:42:709] info: Store: SAVE_WINDOW_SETTINGS
{
  "isMaximized": false,
  "position": [
    574,
    106
  ],
  "size": [
    1024,
    768
  ]
}
[06/27/18, 22:55:47:449] info: Store: FOREGROUND_APP
[06/27/18, 22:55:47:501] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:47:509] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:47:519] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:47:524] info: Store: MAIN_WINDOW_FOCUSED
[06/27/18, 22:55:48:488] info: Store: SAVE_WINDOW_SETTINGS
{
  "isMaximized": false,
  "position": [
    575,
    131
  ],
  "size": [
    1024,
    768
  ]
}
[06/27/18, 22:55:50:244] info: Store: QUIT_APP
[06/27/18, 22:55:50:257] info: Store: UPDATE_TEAM_USAGE
{
  "T031FNTUT": 21539470,
  "TB47MRNBZ": 15691
}
[06/27/18, 22:55:50:299] info: Store: REMOVE_WINDOW 2
[06/27/18, 22:55:50:309] info: Store: QUIT_APP
[06/27/18, 22:55:50:313] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:50:359] info: Store: QUIT_APP
